### ISSUE TITLE
Unable to play clips - Firefox claims media is corrupted

### ISSUE THREAD

**Issue by Synth9617** [ShaneIsrael/fireshare#222]
**Describe the bug**
Unable to play any clips

**To Reproduce**
1. Try to play a video?

**Expected behavior**
The Clip to play.

**Screenshots**
Error displayed:
![image](https://github.com/ShaneIsrael/fireshare/assets/32795625/5605abbf-2adb-471f-85d9-eec8e2848148)

AMD Software Settings (What i get my clips from):
![image](https://github.com/ShaneIsrael/fireshare/assets/32795625/4ec10d21-abcf-4a2a-bec4-7f6eaad9b0d0)

**Desktop (please complete the following information):**
 - Client(s) Firefox, Chrome, Opera. All on my Win11 PC. Opera also tested on my Win10 laptop

(Note: IDK why it assigned yall immediately, if this wasnt meant to happen I do apologise)

**Comment by Synth9617** [comment]
Seems like an issue with AVC and Firefox as they now play on Chrome just fine (after putting Cloudflare to dev mode).

TLDR: Firefox hates me. Cloudflare Dev mode broke it for Chromium. Will mark as closed, sorry to waste time
